cardano-api/src/Cardano/Api/Error.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE ExistentialQuantification #-}
 {-# LANGUAGE GADTSyntax #-}
+{-# LANGUAGE DeriveFunctor #-}
```

Comment:
Why isn't your `stylish-haskell` not automatically sorting this?

Reply:
I temporarily disabled it to make a PR that changes `bench` code and forgot to turn it back on for this PR.